### ISSUE TITLE
Fatal error during update notification

### ISSUE THREAD

**Issue by uok** [skyverge/woocommerce-shipping-estimate#18]
WooCommerce informs me that there are updates available. 
When I got to /wp-admin/update-core.php I get the error

`Fatal error: Uncaught Error: Cannot use object of type stdClass as array in /var/www/wordpress/wp-admin/update-core.php:285 Stack trace: #0 /var/www/wordpress/wp-admin/update-core.php(698): list_plugin_updates() #1 {main} thrown in /var/www/wordpress/wp-admin/update-core.php on line 285
`

There are 7 updates available, but only a few of them are displayed before this error line. So I disabled each plugin one by one (renaming the plugin folder in FTP). This plugin seems to be the problem. The error started with the release of new version and happens in both my systems (development and production)

I tried to catch backtrace

```
/var/www/wordpress/wp-admin/update-core.php:698 calling list_plugin_updates()
/var/www/wordpress/wp-admin/update-core.php:280 calling _get_plugin_data_markup_translate()
/var/www/wordpress/wp-admin/includes/plugin.php:162 calling translate()
/var/www/wordpress/wp-includes/l10n.php:182 calling apply_filters()
/var/www/wordpress/wp-includes/plugin.php:208 calling apply_filters()
/var/www/wordpress/wp-includes/class-wp-hook.php:288 calling {closure}()
```

**Comment by bekarice** [comment]
hey @uok sorry we missed the initial ping on this! could you please try a manual update for now, then let me know if you see this on the next release? 

To manually update, you can:

- download the [v2.3.0 zip here](https://github.com/skyverge/woocommerce-shipping-estimate/releases/tag/2.3.0)
- deactivate and delete the plugin on your site (you won't lose any settings)
- upload and activate the v2.3.0 zip file

We'll be including a WooCommerce 3.8 update soon, so we should see the updater notification then again to determine whether this has been patched in our updater code or not (I'm pretty sure we resolved it a bit ago). Thanks!

**Comment by unfulvio** [comment]
Hi there,

This should have been fixed and by using the latest version (2.4.0) shouldn't incur in the problem. Please let us know if you are still experiencing it. 
Cheers
